lms/djangoapps/certificates/views/webview.py

```diff
@@ -97,7 +97,7 @@ def _update_certificate_context(context, user_certificate, platform_name):
 
     # Translators:  The format of the date includes the full name of the month
     context['certificate_date_issued'] = _('{month} {day}, {year}').format(
-        month=user_certificate.modified_date.strftime("%B"),
+        month=_(user_certificate.modified_date.strftime("%B")),
```

Comment:
According to https://docs.djangoproject.com/en/1.11/topics/i18n/translation/#standard-translation:

> The caveat with using variables or computed values, as in the previous two examples, is that Django’s translation-string-detecting utility, django-admin makemessages, won’t be able to find these strings. More on makemessages later.

In other words, it will 'work' so long as these strings appear somewhere else in the code, as they do here: https://github.com/edx/edx-platform/blob/master/common/djangoapps/util/date_utils.py#L344. I think it would be better if we could make use of those utils here (or that dictionary of month names) instead of relying on the month names to be extracted/translated somewhere else in the code.